app/controllers/service_providers_controller.rb

```diff
@@ -167,17 +167,6 @@ def cache_logo_info
     service_provider.remote_logo_key = service_provider.logo_file.key
   end
 
-  def using_s3?
```

Comment:
It looks like all the storage config is happening in `config/storage.tml` ([here](https://github.com/18F/identity-dashboard/blob/main/config/storage.yml)) so this code can be removed